paddle/fluid/platform/dynload/dynamic_loader.cc

```diff
@@ -172,23 +172,28 @@ static inline void* GetDsoHandleFromSearchPath(
   // 5. [If Failed] logging or throw error info
   if (nullptr == dso_handle) {
     auto error_msg =
-        "Failed to find dynamic library: %s ( %s ) \n"
-        "Please specify its path correctly using following ways: \n"
-        "  set environment variable LD_LIBRARY_PATH on Linux or "
-        "DYLD_LIBRARY_PATH on Mac OS. \n"
-        "  For instance, issue command: export LD_LIBRARY_PATH=... \n"
-        "  Note: After Mac OS 10.11, using the DYLD_LIBRARY_PATH is "
-        "impossible unless System Integrity Protection (SIP) is disabled.";
+        "The third-party library (%s) that Paddle depends on is not "
+        "configured correctly. Failed to find dynamic library: %s ( %s )\n"
+        "  Suggestions:\n"
+        "  1. Check if the third-party library is installed correctly.\n"
```

Comment:
can it be 
" Check if 3rd dynamic library (e.g. CUDA, CUDNN) is installed correctly and version is matched with Paddle. "

Reply:
done, thx